samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md

```diff
@@ -32,7 +32,7 @@ Test serialization of outer boolean types
 // The following code samples are still beta. For any issue, please report via http://github.com/OpenAPITools/openapi-generator/issues/new
 import PetstoreClient
 
-let body = false // Bool | Input boolean as post body (optional)
+let body = true // Bool | Input boolean as post body (optional)
```

Comment:
@4brunu I notice that the value has been flipped from `true` to `false` int the sample code. Is that expected?

Reply:
I was also surprised.
That happen after running the pet projects. 
The input spec has changed, but I didn't change it.

Reply:
👌

Reply:
I'll take another look after merging this PR instead.

Reply:
Looks like that's the correct behavior: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L1421 (and it was fixed somehow)